src/ResourceManager/Profile/Commands.Profile.Test/ProfileController.cs

```diff
@@ -13,63 +13,44 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Commands.Common.Authentication;
-using Microsoft.Azure.Internal.Subscriptions;
-using Microsoft.Azure.Management.ResourceManager;
 using Microsoft.Azure.ServiceManagemenet.Common.Models;
 using Microsoft.Azure.Test.HttpRecorder;
 using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
-using Microsoft.WindowsAzure.Commands.Common;
 using Microsoft.WindowsAzure.Commands.Common.Test.Mocks;
 using Microsoft.WindowsAzure.Commands.ScenarioTest;
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using LegacyTest = Microsoft.Azure.Test;
+using Microsoft.Azure.Management.Internal.Resources;
+using SubscriptionClient = Microsoft.Azure.Internal.Subscriptions.SubscriptionClient;
 
 namespace Microsoft.Azure.Commands.Resources.Test.ScenarioTests
 {
     public sealed class ProfileController
     {
-        private EnvironmentSetupHelper helper;
-        private const string TenantIdKey = "TenantId";
-        private const string DomainKey = "Domain";
-        private const string SubscriptionIdKey = "SubscriptionId";
+        private readonly EnvironmentSetupHelper _helper;
 
+        public SubscriptionClient SubscriptionClient { get; private set; }
 
-        public Internal.Subscriptions.SubscriptionClient SubscriptionClient { get; private set; }
+        public ResourceManagementClient NewResourceManagementClient { get; private set; }
```

Comment:
Have we removed one version of resource management client? I wonder if this might have to do with completer faliures.

Reply:
I believe there might have been a version of the ResourceManager client that was being used by this that might be causing the issue above. Again, not 100% sure here. As of this PR, all test controllers use our Internal ResourceManager client (except for Resources, of course). We'll need to revisit this. Let me make an issue.

Reply:
Created an issue: https://github.com/Azure/azure-powershell/issues/7990